server/src/test/java/org/elasticsearch/index/IndexingSlowLogTests.java

```diff
@@ -430,18 +430,18 @@ public void testSetLevels() {
 
         metadata = newIndexMeta("index", Settings.builder().put(IndexMetadata.SETTING_VERSION_CREATED, IndexVersion.current()).build());
         settings.updateIndexMetadata(metadata);
-        assertEquals(TimeValue.timeValueMillis(-1).nanos(), log.getIndexTraceThreshold());
-        assertEquals(TimeValue.timeValueMillis(-1).nanos(), log.getIndexDebugThreshold());
-        assertEquals(TimeValue.timeValueMillis(-1).nanos(), log.getIndexInfoThreshold());
-        assertEquals(TimeValue.timeValueMillis(-1).nanos(), log.getIndexWarnThreshold());
+        assertEquals(TimeValue.timeValueNanos(-1).nanos(), log.getIndexTraceThreshold());
```

Comment:
That's an awfully-long-winded way to say `-1` :)

```suggestion
        assertEquals(-1L, log.getIndexTraceThreshold());
```

Reply:
Using `TimeValue.MINUS_ONE` as the default for the settings requires `-1000000` here.  I switched to `TimeValue.MINUS_ONE.nanos()`.